.github/actions/deploy-happy-stack/action.yml

```diff
@@ -1,8 +1,12 @@
 name: Create or Update a Happy Stack
-description: "Will create or update a Happy Stack"
+description: "Will create, update, or delete a Happy Stack"
 inputs:
+  operation:
+    description: "Operation we want this to perform. Defaults to create-or-update. Set to delete if you want to delete the stack."
```

Comment:
maybe change the default description to "If not provided, happy will automatically create or update the stack with given name"

Reply:
Just updated!